src/devices/CharacterLcd/Lcd1602.cs

```diff
@@ -35,8 +35,18 @@ public Lcd1602(int registerSelectPin, int enablePin, int[] dataPins, int backlig
         /// This is for on-chip I2c support. For connecting via I2c GPIO expanders, use the GPIO constructor <see cref="Lcd1602(int, int, int[], int, float, int, GpioController)"/>.
         /// </remarks>
         /// <param name="device">The I2c device for the LCD.</param>
-        public Lcd1602(I2cDevice device)
-            : base(new Size(16, 2), LcdInterface.CreateI2c(device))
+        /// <param name="uses8Bit">True if the device uses 8 Bit commands, false if it handles only 4 bit commands.</param>
+        public Lcd1602(I2cDevice device, bool uses8Bit)
+            : base(new Size(16, 2), LcdInterface.CreateI2c(device, uses8Bit))
+        {
+        }
+
+        /// <summary>
+        /// Constructs a new 16x2 LCD controller with the given interface
+        /// </summary>
+        /// <param name="interface">The LCD Interface</param>
+        public Lcd1602(LcdInterface @interface)
```

Comment:
can you use `lcdInterface` to avoid `@`

Reply:
Ok. The existing code was like this as well, but I can change that.

Reply:
usually it's fine to leave if it matches the existing style (you don't need to fix all the comments I write if you feel it doesn't make sense - I might miss some details or lack some context when reviewing). Since you've already changed I like the new version better :)